spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1539,7 +1539,7 @@ For Spring Cloud Stream samples, please refer to the https://github.com/spring-c
 == Getting Started
 
 To get started with creating Spring Cloud Stream applications, visit the https://start.spring.io[Spring Initializr] and create a new Maven project named "GreetingSource".
-Select Spring Boot version 1.3.4 SNAPSHOT and search or tick the checkbox for Stream Kafka (we will be using Kafka for messaging).
+Select the Spring Boot version that works for you and search or tick the checkbox for Stream Kafka (we will be using Kafka for messaging).
```

Comment:
So 
`:spring-boot-supported-versions: 1.3.x and 1.4.x`

and

Note that Spring Cloud Stream applications are known to work with Spring Boot versions {spring-boot-supported-versions}

?